Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -743,7 +743,7 @@ final class PackageToolTests: CommandsTestCase {
             XCTAssertMatch(contents, .prefix("// swift-tools-version:\(version < .v5_4 ? "" : " ")\(versionSpecifier)\n"))
 
             XCTAssertFileExists(manifest)
-            XCTAssertEqual(try fs.getDirectoryContents(path.appending(component: "Sources")), ["CustomName.swift"])
+            XCTAssertEqual(try fs.getDirectoryContents(path.appending(component: "Sources")), ["main.swift"])
```

Comment:
is this test testing anything at this point?

Reply:
It is pretty similar to the base case at this point. I was thinking we want to keep it since we'll likely want to revisit this in the future.

Reply:
okay